openmetadata-ui/src/main/resources/ui/playwright/constant/entity.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ *  Copyright 2024 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+export const ES_RESERVED_CHARACTERS: Record<string, string> = {
+  '+': '\\+',
+  '-': '\\-',
+  '=': '\\=',
+  '&': '\\&',
+  '&&': '\\&&',
+  '||': '\\||',
+  '>': '\\>',
+  '<': '\\<',
+  '!': '\\!',
+  '(': '\\(',
+  ')': '\\)',
+  '{': '\\{',
+  '}': '\\}',
+  '[': '\\[',
+  ']': '\\]',
+  '^': '\\^',
+  '"': '\\"',
+  '~': '\\~',
+  '*': '\\*',
+  '?': '\\?',
+  ':': '\\:',
+  '\\': '\\\\',
```

Comment:
you can directly use this https://github.com/open-metadata/OpenMetadata/blob/dc31717b169fe374af3a40c358ab3b7f9500126f/openmetadata-ui/src/main/resources/ui/cypress/constants/EntityConstant.ts#L487

Reply:
I've kept it separate since we do the same for the functionality code to keep it clean. Expanding or reusing somewhere else in the future will be easier if needed.